library/pod-security-policy/users/template.yaml

```diff
@@ -102,8 +102,17 @@ spec:
 
         get_user_violation(params, container) = msg {
           not get_field_value("runAsUser", container, input.review)
+          not get_field_value("runAsNonRoot", container, input.review)
           params.rule != "RunAsAny"
-          msg := sprintf("Container %v is attempting to run without a required securityContext/runAsUser. Allowed runAsUser: %v", [container.name, params])
+          params.rule != "runAsNonRoot"
+          msg := sprintf("Container %v is attempting to run without a required securityContext/%v", [container.name, params.rule])
+        }
+
+        get_user_violation(params, container) = msg {
+          params.rule = "runAsNonRoot"
```

Comment:
I think this should be `MustRunAsNonRoot`?

Reply:
Fixed